docs/web/diff.md

```diff
@@ -0,0 +1,903 @@
+  ----=================----
+  ```
+* Show outstanding reports that disappeared from first analysis to the next (via `--resolved`): 
+  * Local-local diff in between `./test_report_dir` to `./test_report_dir_updated/`:
+    ```sh
+    CodeChecker cmd diff --basename ./test_report_dir --newname ./test_report_dir_updated/ --resolved
+    ```
+  * Remote-remote diff in between `Test Run` and `Test Run Updated`:
+    ```sh
+    CodeChecker cmd diff --basename "Test Run" --newname "Test Run Updated" --resolved
+    ```
+  * Remote-local or local-remote diff:
+    ```sh
+    CodeChecker cmd diff --basename "Test Run" --newname ./test_report_dir_updated/ --resolved
+    CodeChecker cmd diff --basename ./test_report_dir --newname "Test Run Updated" --resolved
+    ```
+  Expected output:
+  ```
+  ----==== Checker Statistics ====----
+  ----------------------------------------------
+  Checker name    | Severity | Number of reports
+  ----------------------------------------------
+  core.DivideZero | HIGH     |                 1
+  ----------------------------------------------
+  ----=================----
+  ```
+* Show outstanding reports that remained from first analysis to the next (via `--unresolved`): 
+  * Local-local diff in between `./test_report_dir` to `./test_report_dir_updated/`:
+    ```sh
+    CodeChecker cmd diff --basename ./test_report_dir --newname ./test_report_dir_updated/ --unresolved
+    ```
+  * Remote-remote diff in between `Test Run` and `Test Run Updated`:
+    ```sh
+    CodeChecker cmd diff --basename "Test Run" --newname "Test Run Updated" --unresolved
+    ```
+  * Remote-local or local-remote diff:
+    ```sh
+    CodeChecker cmd diff --basename "Test Run" --newname ./test_report_dir_updated/ --unresolved
+    CodeChecker cmd diff --basename ./test_report_dir --newname "Test Run Updated" --unresolved
+    ```
+  Expected output:
+  ```sh
+  CodeChecker cmd diff --basename ./test_report_dir --newname ./test_report_dir_updated/ --unresolved
+  ```
+  ```
+  ----==== Checker Statistics ====----
+  --------------------------------------------------
+  Checker name        | Severity | Number of reports
+  --------------------------------------------------
+  deadcode.DeadStores | LOW      |                 1
+  --------------------------------------------------
+  ----=================----
+  ```
+
+### Using the web GUI
+
+Select "Test Run" to be in the "baseline" set by clicking on the left radio button on the right hand side. Select "Test Run Updated" to be in the "newline" set by clicking on the other one. Click on "DIFF" in the upper right corner.
+
+![image](https://github.com/Szelethus/codechecker/assets/23276031/75490051-86df-419d-998a-ec89942faca1)
+
+In the middle of the image, we can see the list of oustadning reports that appeared from the baseline to the newline. In the bottom left corner, by clicking on the cogwheel next to "Diff type", you can change diff type:
```

Comment:
The steps of the operation could be highlighted on the GUI images, e.g. with a square or an arrow.